src/py/_pyodide/_core_docs.py

```diff
@@ -273,7 +273,7 @@ def from_file(self, file: IOBase, /):
         data once.
         """
 
-    def _into_file(self, file: IOBase, /):
+    def _into_file(self, file: IOBase, /) -> Any:
```

Comment:
https://github.com/pyodide/pyodide/blob/af972267c7969c6f023da709851463ec99e7e4ac/src/core/jsproxy.c#L1762
```suggestion
    def _into_file(self, file: IOBase, /) -> None:
```